src/views/Home/index.tsx

```diff
@@ -1,36 +1,98 @@
-import { Box, Link, Typography } from '@mui/material'
+import { Box, Fade, Link, Typography } from '@mui/material'
 import { Stack } from '@mui/system'
-
+import { FileUploadOutlined as UploadIcon } from '@mui/icons-material'
 import { useIntl } from 'react-intl'
+import { DropzoneInputProps, DropzoneRootProps, useDropzone } from 'react-dropzone'
+import { useCallback, useState } from 'react'
 import { AddMapButton } from '../../components/AddMapButton'
 import { AuthorisedLayout } from '../../layouts/Authorised'
-import { Img } from './styles'
+import { Img, Overlay } from './styles'
 import { messages as msgs } from './messages'
+import { unzip } from '../../helpers/file'
+import { Loader } from '../../components/Loader'
+import { useCreateMap } from '../../hooks/useCreateMap'
+
+export const HomeView = () => {
+  const {
+    createMap,
+    progress: { completed, loading, error, done },
+  } = useCreateMap()
+
+  const onDrop = useCallback(async (acceptedFiles: File[]) => {
+    if (!acceptedFiles.length || !acceptedFiles[0].name.match(/.mapeomap$/))
+      return console.log('invalid file', acceptedFiles[0])
+    const unzippedFiles = await unzip(acceptedFiles[0])
+    console.log({ unzippedFiles })
+    const data = createMap(unzippedFiles)
+  }, [])
+
+  const { getRootProps, getInputProps, open, isDragActive } = useDropzone({
+    noClick: true,
+    noKeyboard: true,
+    accept: ['.mapeomap'],
+    onDrop,
+  })
+
+  return (
+    <AuthorisedLayout onClickAddMap={open}>
+      {done || loading ? (
```

Comment:
should this be: `!done || loading`?

Reply:
Oh I see what s going on. When the user first logs in `done===false`, but when they upload a map, `done===true`. With the current logic, they see the `NoMaps` when they first log in, but as soon as they complete uploading 1 map it is a blank screen. I think we can solve this by getting rid of the done state completely